packages/engine/Source/Scene/Cesium3DTilesVoxelProvider.js

```diff
@@ -47,54 +69,184 @@ import VoxelShapeType from "./VoxelShapeType.js";
 function Cesium3DTilesVoxelProvider(options) {
   options = defaultValue(options, defaultValue.EMPTY_OBJECT);
 
-  /** @inheritdoc */
-  this.shapeTransform = undefined;
-
-  /** @inheritdoc */
-  this.globalTransform = undefined;
-
-  /** @inheritdoc */
-  this.shape = undefined;
-
-  /** @inheritdoc */
-  this.minBounds = undefined;
+  const {
+    className,
+    names,
+    types,
+    componentTypes,
+    shape,
+    dimensions,
+    paddingBefore = Cartesian3.ZERO.clone(),
+    paddingAfter = Cartesian3.ZERO.clone(),
+    globalTransform = Matrix4.IDENTITY.clone(),
+    shapeTransform = Matrix4.IDENTITY.clone(),
+    minBounds,
+    maxBounds,
+    minimumValues,
+    maximumValues,
+    maximumTileCount,
+  } = options;
 
-  /** @inheritdoc */
-  this.maxBounds = undefined;
-
-  /** @inheritdoc */
-  this.dimensions = undefined;
-
-  /** @inheritdoc */
-  this.paddingBefore = undefined;
-
-  /** @inheritdoc */
-  this.paddingAfter = undefined;
-
-  /** @inheritdoc */
-  this.names = undefined;
-
-  /** @inheritdoc */
-  this.types = undefined;
-
-  /** @inheritdoc */
-  this.componentTypes = undefined;
-
-  /** @inheritdoc */
-  this.minimumValues = undefined;
-
-  /** @inheritdoc */
-  this.maximumValues = undefined;
+  //>>includeStart('debug', pragmas.debug);
+  Check.typeOf.object("names", names);
+  Check.typeOf.object("types", types);
+  Check.typeOf.object("componentTypes", componentTypes);
+  Check.typeOf.string("shape", shape);
+  Check.typeOf.object("dimensions", dimensions);
+  //>>includeEnd('debug');
 
-  /** @inheritdoc */
-  this.maximumTileCount = undefined;
+  /**
+   * A transform from shape space to local space. If undefined, the identity matrix will be used instead.
```

Comment:
```suggestion
   * A transform from shape space to local space.
```

Reply:
The `undefined` case is handled in the constructor, but this property should never be `undefined`.